src/Agent.Plugins/PipelineCache/RestorePipelineCacheV0.cs

```diff
@@ -10,23 +12,20 @@ public class RestorePipelineCacheV0 : PipelineCacheTaskPluginBase
         public override string Stage => "main";
 
         protected override async Task ProcessCommandInternalAsync(
-            AgentTaskPluginExecutionContext context, 
-            string path, 
-            string keyStr,
-            string salt,
+            AgentTaskPluginExecutionContext context,
```

Comment:
Can we create a new version for this change?

Reply:
I think we decided against - is that right?

---
In reply to: [306960674](https://github.com/microsoft/azure-pipelines-agent/pull/2358#discussion_r306960674) [](ancestors = 306960674)